libcrux-ml-dsa/README.md

```diff
@@ -1,10 +1,10 @@
 # ML-DSA
 
```

Comment:
This needs a verification label.

Reply:
looks good here. Let's make sure we have the same structure here as for ml-kem (see comment there).

Reply:
I made the formats uniform. One last look from you and we can merge?